api-reference/webhooks.mdx

```diff
@@ -110,7 +110,7 @@ The request body is a `JSON` object with the fields below.
 
 The JSON schema for Plain the webhook request body can be found [here](https://core-api.uk.plain.com/webhooks/schema.json).
 
-If you prefer, you can browse the schema using [this interactive tool](https://json-schema.app/view/%23?url=https%3A%2F%2Fcore-api.uk.plain.com%2Fwebhooks%2Fschema.json).
+If you prefer, you can browse the schema using [this interactive tool](https://core-api.uk.plain.com/webhooks/schema.json).
```

Comment:
Can we also update the text here? Maybe:
> If you prefer, you can browse the schema [here](...).

Reply:
Oh shit, good point!

Reply:
I actually just removed that line as we link the schema directly above